patches/engine/0001-snappy-buildkit-git-environ.patch

```diff
@@ -0,0 +1,27 @@
+From 65d421326d7edf6a96ce7b62d0c216d1b59e6509 Mon Sep 17 00:00:00 2001
+From: Lincoln Wallace <lincoln.wallace@canonical.com>
+Date: Fri, 30 Aug 2024 11:52:32 -0300
+Subject: [PATCH 1/1] snappy buildkit git environ
+
+Signed-off-by: Lincoln Wallace <lincoln.wallace@canonical.com>
+---
+ vendor/github.com/moby/buildkit/util/gitutil/git_cli.go | 3 +++
+ 1 file changed, 3 insertions(+)
+
+diff --git a/vendor/github.com/moby/buildkit/util/gitutil/git_cli.go b/vendor/github.com/moby/buildkit/util/gitutil/git_cli.go
+index 5c35f9365b..b0e0fb3aef 100644
+--- a/vendor/github.com/moby/buildkit/util/gitutil/git_cli.go
++++ b/vendor/github.com/moby/buildkit/util/gitutil/git_cli.go
+@@ -183,6 +183,9 @@ func (cli *GitCLI) Run(ctx context.Context, args ...string) (_ []byte, err error
+ 
+ 		cmd.Env = []string{
+ 			"PATH=" + os.Getenv("PATH"),
++      "LD_LIBRARY_PATH=" + os.Getenv("LD_LIBRARY_PATH"),
++      "GIT_EXEC_PATH=" + os.Getenv("GIT_EXEC_PATH"),
++      "GIT_TEMPLATE_DIR=" + os.Getenv("GIT_TEMPLATE_DIR"),
```

Comment:
Inconsistent indentation.

Reply:
This is tricky! I tried a few times to get this indentation right, but once the patch is formatted/generated it didn't get right. And manually edditing the patch seems wrong/hacking for me.

But since this is GO and not Python, I don't see a problem here.

Reply:
So Go code doesn't need to be indented because the compiler doesn't care? :)